### PR TITLE
Added Ids column in Policies and Roles List

### PR DESCRIPTION
**Pull request by samshinde**
Signed-off-by: samshinde <ashinde@chef.io>

### :nut_and_bolt: Description: What code changed, and why?
In the Roles and policy list tables added a column for ID.

### :chains: Related Resources
#4036 
### :+1: Definition of Done
Added a column for ID on the policy list and role list pages.
### :athletic_shoe: How to Build and Test the Change
```
STEP 1
inside the hab studio

[default:/src:0]# build components/automate-ui-devproxy/
[default:/src:0]# start_automate_ui_background
[default:/src:0]# start_all_services

STEP 2
open new window
go to automate UI path

$ cd components/automate-ui
and run the command ```
### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable

### CODE REVIEW

**Review by srenatus — APPROVED** [review]
LGTM

**Comment by srenatus** [comment]
⚠️ There's some tests to update: https://buildkite.com/chef-oss/chef-automate-master-verify/builds/14162#9f518c15-6214-4bdd-8edc-78c5a98061f9/84-706

**Review by srenatus — CHANGES_REQUESTED** [review]
Sorry this is the wrong way to fix the tests, I'm afraid. Can you alter the assertions to match what's in the policy list, and add one for the policy IDs to be displayed, please?

**Review by msorens — COMMENTED** [review]
Also, as I mentioned several days ago, it is important not to squash your changes into a single commit, especially when making changes responding to reviewer feedback.

**Comment by samshinde** [comment]
@srenatus @msorens Could you please help me out on this as I have done the primary analysis but we didn't find the file which will solve the error. If you have any opinion on that, please suggest.

File: components/automate-ui/e2e/admin.e2e-spec.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ describe('Admin pages', () => {
           {
             policies: [
               {
-                id: 'some-policy-id',
+                id: 'Custom',
```
Comment (srenatus) [inline components/automate-ui/e2e/admin.e2e-spec.ts:121]: Please leave theses as-is. The "Custom" and "Chef-managed" are the **types** of the policies, as displayed in the UI, and as asserted by the the tests.

File: components/automate-ui/src/app/modules/policy/list/policy-list.component.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
                 <chef-thead>
                   <chef-tr>
                     <chef-th class="name-column">Name</chef-th>
+                    <chef-th class="id-column">ID</chef-th>
```
Comment (srenatus) [inline components/automate-ui/src/app/modules/policy/list/policy-list.component.html:16]: We're adding a new column here, so we need to adapt the `nth` number used for selection table cells here: 
https://github.com/chef/automate/blob/9e32a4ca5c2966e7518a75ea67dc2f534473641c/components/automate-ui/e2e/admin.e2e-spec.ts#L167-L170

Try `chef-td:nth-child(3)` instead, and add an assertion for the newly added column, like

```typescript
 waitForElement('chef-table chef-tr:nth-child(1) chef-td:nth-child(2)') 
   .then((policyType) => { 
     expect(policyType.getText()).toBe('chef-managed-administrator'); 
   }); 
```

**Review by srenatus — CHANGES_REQUESTED** [review]
Added a note on how to adapt the tests! :)

File: components/automate-ui/src/app/modules/policy/list/policy-list.component.html (hunk 2 above)
Comment (samshinde) [reply components/automate-ui/src/app/modules/policy/list/policy-list.component.html:16]: @srenatus  I tried these commands
``` 
ng e2e --no-webdriver-update
sudo make e2e
```
for resolving binaries error:
```
sudo webdriver-manager update
```
after code changes to test code locally but it gives me error as in below attachment and not able to test the changes locally.

test error

Comment (msorens) [reply components/automate-ui/src/app/modules/policy/list/policy-list.component.html:16]: This question is unrelated to the PR; if still having difficulty you should post this question in the #ui-team channel.

**Comment by msorens** [comment]
This PR still needs to add tests for the ID column as srenatus indicated above.

**Review by msorens — APPROVED** [review]
Looks like the requested test changes have been done, so I am approving, but since this is so old, though, I think it would be prudent to rebase before merging to see if all the tests are still good.

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**